hack/check-commits.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+parent_branch="master"
+git remote add openshift https://github.com/openshift/assisted-installer.git &> /dev/null || true
+git fetch openshift ${parent_branch}
+
+current_branch="$(git rev-parse --abbrev-ref HEAD)"
+
+revs=$(git rev-list "FETCH_HEAD".."${current_branch}")
```

Comment:
Why close the quotations after FETCH_HEAD? all the argument to `rev-list` should be a single string.
```suggestion
revs=$(git rev-list "FETCH_HEAD..${current_branch}")
```